.pfnci/matrix.yaml

```diff
@@ -730,26 +730,26 @@
 # CUDA Python Test
 ###############################################################################
 
-# CUDA 11.x / CUDA Python | Linux
-- project: "cupy.linux.cuda11x-cuda-python"
-  target: "cuda11x-cuda-python"
+# CUDA 12.x / CUDA Python | Linux
+- project: "cupy.linux.cuda12x-cuda-python"
```

Comment:
@kmaehashi I need to move the cuda-python CI to 12.x because we need a new NVRTC API from 12.x. But it seems a new `cupy.linux.cuda12x-cuda-python` pipeline cannot be registered by making changes in the codebase? I assume there's a backend that need to be updated as well? 🙂

Reply:
Thanks, Kenichi-san! I'll rerun the CI after 13.6.0 is out, to avoid resource contention.